DOCS-8686 Workday 12.1  ETA 8/31 (#957)
mulesoft/docs-connectors: workday/12.1/modules/ROOT/pages/index.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for Workday (Workday Connector) provides access to standard Workday operations. Workday is a Software-as-a-Service (SaaS) solution developed by Workday, Inc.
Workday provides a set of software apps for financial and human capital management, such as human resources, staffing, recruiting, and performance.

Workday Connector provides operations that you can use to perform tasks for educational organizations, human resources, professional services, payroll, and benefits administration.

For compatibility information and fixed issues, see the release notes.

== Prerequisites

Before creating an app, you must have access to the Workday target resource and
Anypoint Platform. You also need to understand how to create a Mule app using Anypoint Studio.

== Audience

* New users:

** To create your Mule app, read xref:workday-connector-studio.adoc[Studio Configuration].
** The xref:workday-connector-examples.adoc[Examples] topic provides one or more use cases for using the connector.

> * Advanced users, read the following:

Reviewer: This is not worded or formatted like its peer above?

** xref:workday-connector-xml-maven.adoc[XML and Maven Support]
** xref:workday-connector-examples.adoc[Examples]

> == Common Use Cases For the Connector

Reviewer: Fix template too, if this is part of it:
```suggestion
== Common Use Case
```

> The common use case for Workday Connector is starting any of Workday's standard operations. The xref:workday-connector-examples.adoc[Example] use cases include creating a position request in Workday or adding a fund to the financial management service.

Reviewer: Consider active voice:
```suggestion
Use Workday Connector to start any of Workday's standard operations, such as creating a position request in Workday or adding a fund to a financial management service.
```

> == Using Exchange Templates and Examples

Reviewer: ```suggestion
== List of Exchange Templates and Examples
```
This is really just a reference topic.

> https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates

Reviewer: ```suggestion
https://www.mulesoft.com/exchange/[Anypoint Exchange] provides templates that
```

> you can use as a starting point for your app, as well as examples that illustrate a complete solution.

Reviewer: ```suggestion
you can use as starting points for your apps and examples that illustrate complete solutions:
```

* https://www.mulesoft.com/exchange/org.mule.templates/template-wday2sfdc-worker-migration[Salesforce to Workday Financials Account - Customer Migration]
* https://anypoint.mulesoft.com/exchange/org.mule.templates/template-wday2snow-employee-aggregation[Workday and ServiceNow Employee Aggregation]
* https://www.mulesoft.com/exchange/org.mule.templates/template-wday2sfdc-worker-aggregation[Workday and Salesforce Worker Aggregation]
* https://anypoint.mulesoft.com/exchange/org.mule.templates/template-wday2snow-worker-migration[Workday to ServiceNow Service Worker Migration]
* https://anypoint.mulesoft.com/exchange/org.mule.templates/template-wday2snow-worker2servicerequest-broadcast[Workday Worker to ServiceNow Service Request Broadcast]
* https://anypoint.mulesoft.com/exchange/org.mule.templates/template-wday2snow-worker2user-broadcast[Workday Worker to ServiceNow User Broadcast]
* https://www.mulesoft.com/exchange/org.mule.templates/template-wday2sfdc-workercase-broadcast[Workday to Salesforce Worker Case Broadcast]

== Next Step

> After you complete the prerequisites and experiment with templates and examples, you are ready to create an app with

Reviewer: Would this be a good template addition?
```suggestion
After you complete the prerequisites and experiment with templates and examples, you are ready to create your own app with
```

Author: This is already part of the template.
